module/skill/GetUp/data/config/GetUp.yaml

```diff
@@ -1,6 +1,8 @@
 # Controls the minimum log level that NUClear log will display
 log_level: INFO
 
+delay_time: 500 #  milliseconds
+
 scripts:
   getup_upright:
     - "Stand.yaml"
```

Comment:
Just below this is a getup left and getup right with just stand up front script running. It might be better to change the logic to just wait in these instances rather than run any script, as we do end up on front or back when it falls to the side.